docs/manual/annotating-libraries.tex

```diff
@@ -510,6 +510,37 @@
 since the value is known to be a key for all three maps.
 
 
+\subsectionAndLabel{Stub methods in subclasses of the declaring class}{stub-fake-override}
+
+Sometimes, a method's return type is different in a subclass than in a
+superclass, even though the subtype does not define the method and just
+inherits the superclass's definition.
+
+You may write a ``fake override'' in a stub file.  This is a declaration of
+a method that is inherited.  The Java signature of the method must be
+identical to that in the superclass where it is actually defined, but you
+can change its annotations.
```

Comment:
If the "fake override" doesn't have any explicit annotations, is it as if the "fake override" wasn't written?

For example,
```
class Super{
  @Nullable String method() {... }
}

class Sub extends Super {}

class Use {
  void use(Sub sub) {
    @NonNull String s = sub.method();  // Is this legal?
  }
}
```
```
// In a stub file:
class Sub extends Super {
   String method(); 
}
```